Files/cart/utils.py

```diff
@@ -61,6 +80,7 @@ def GetCart(user_id):
     response={"result":response,"total_cart_price":total_cart_price}
     return response
 
+@cors
```

Comment:
We've added a cross_origin decorator in routes.py
Adding @cors here won't conflict with that right?